### PR TITLE
gitg 41 libgit2-glib 1.0.0.1

### LINKED ISSUE #92229: gitg 41
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
$ brew config
HOMEBREW_VERSION: 3.3.9-80-g26694fd
ORIGIN: https://github.com/Homebrew/brew
HEAD: 26694fd6354c52cfc592261c86752ca4fc9e3b3d
Last commit: 20 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 1c1a3952e33a8d2df1522b5cfd9b0f9d08801fa1
Core tap last commit: 15 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: octa-core 64-bit skylake
Clang: 13.0.0
Git: 2.33.1 => /bin/git
Curl: 7.79.1 => /bin/curl
Kernel: Linux 5.15.8-200.fc35.x86_64 x86_64 GNU/Linux
OS: Fedora release 35 (Thirty Five)
Host glibc: 2.34
/usr/bin/gcc: 11.2.1
/usr/bin/ruby: 3.0.2
glibc: N/A
gcc@5: N/A
xorg: N/A
```

```
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have uncommitted modifications to Homebrew/homebrew-core.
If this is a surprise to you, then you should stash these modifications.
Stashing returns Homebrew to a pristine state but can be undone
should you later need to do so for some reason.
  cd /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core && git stash && git clean -d -f

Uncommitted files:
   M Formula/gitg.rb
   M Formula/libgit2-glib.rb
```
```


### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Bump gitg to recent release 41

### What happened (include all command output)?

Everything build correctly

### What did you expect to happen?

Upgrade gitg to version 41

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
$ brew install libgit2-glib
$ brew install gitg
$ gitg

A review from @tschoonj would be appreciated.
```

### PR DESCRIPTION
**Pull request by albfan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Tested for libgit2-glib and gitg

A review from @tschoonj would be appreciated

fixes #92229

### CODE REVIEW

File: Formula/gitg.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Gitg < Formula
   desc "GNOME GUI client to view git repositories"
   homepage "https://wiki.gnome.org/Apps/Gitg"
-  url "https://download.gnome.org/sources/gitg/3.32/gitg-3.32.1.tar.xz"
-  sha256 "24a4aabf8a42aa2e783e1fb5996ebb3c2a82a01b9689269f1329517ef124ef5a"
+  url "https://download.gnome.org/sources/gitg/41/gitg-41.tar.xz"
+  sha256 "7fb61b9fb10fbaa548d23d7065babd72ad63e621de55840c065ce6e3986c4629"
   revision 7
```
Comment (carlocab) [inline Formula/gitg.rb:6]:
```suggestion
```
`revision`s should be removed at version bumps.

File: Formula/libgit2-glib.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 class Libgit2Glib < Formula
   desc "Glib wrapper library around libgit2 git access library"
   homepage "https://github.com/GNOME/libgit2-glib"
-  url "https://download.gnome.org/sources/libgit2-glib/0.99/libgit2-glib-0.99.0.1.tar.xz"
-  sha256 "e05a75c444d9c8d5991afc4a5a64cd97d731ce21aeb7c1c651ade1a3b465b9de"
-  license "LGPL-2.1"
+  url "https://download.gnome.org/sources/libgit2-glib/1.0/libgit2-glib-1.0.0.1.tar.xz"
+  sha256 "460a5d6936950ca08d2d8518bfc90c12bb187cf6e674de715f7055fc58102b57"
+  license "LGPL-2.1-only"
   revision 4
```
Comment (carlocab) [inline Formula/libgit2-glib.rb:7]:
```suggestion
```

File: Formula/gitg.rb (hunk 1 above)
Comment (albfan) [reply Formula/gitg.rb]: Ah ok

**Review by iMichka — APPROVED** [review]

**Comment by iMichka** [comment]
Thanks @albfan  for your first contribution to homebrew-core !

**Pull request closed without merging**